docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,374 @@
+{
```

Comment:
<p>Add before this a title:</p><p><code>Define the Pipeline</code></p><p> </p><p>text: To define the pipeline we pass the both the run and shape function to the DALI numba operator.</p><p> </p><p>Maybe mention the dims and types that we need to specify.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>